tm2/pkg/bft/blockchain/pool.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/bft/types"
 	"github.com/gnolang/gno/tm2/pkg/flow"
 	"github.com/gnolang/gno/tm2/pkg/log"
-	"github.com/gnolang/gno/tm2/pkg/p2p"
+	types2 "github.com/gnolang/gno/tm2/pkg/p2p"
```

Comment:
The import name seems strange to me. Does it mean something I missed?

Reply:
If this PR can be visualized now 😄 

![istockphoto-154930761-612x612](https://github.com/user-attachments/assets/ad1e4b21-74a9-4fdd-b340-da56491e9096)

It's still a heavy WIP, right now you can't even build correctly since I'm simplifying the imports and types.

I'll ping as soon as it's ready for viewing and testing 🙏